webdriver/tests/bidi/browsing_context/navigate/__init__.py

```diff
@@ -3,23 +3,22 @@
 from webdriver.bidi.error import UnknownErrorException
 
 
-async def navigate_and_assert(bidi_session, context, url, wait="complete", expected_error=False):
+async def navigate_and_assert(bidi_session, context_id, url, wait="complete", expected_error=False):
```

Comment:
Would you mind explaining this change. Why is it necessary. I think that we should keep it in sync with the `top_context` fixture.

Reply:
We already have some discrepancy in using `context_object` like `{context: "CONTEXT_ID"}` vs `context_id` like  `"CONTEXT_ID"`:
1. [`bidi_session.browsing_context.create`](https://github.com/web-platform-tests/wpt/blob/0f90713c40d6a96f9d0f4c92ba36b97c740aa1e3/tools/webdriver/webdriver/bidi/modules/browsing_context.py#L25) returns `context_id`.
2. Fixture `top_context` is a `context_object`.

As long as `bidi_session.browsing_context.create` returns `context_id`, I made the `new_tab` to be a `context_id` as well, as nothing else is used. I'd suggest we don't use `context_object` at all unless it is required for verification, but the change would be quite huge and not related to the current one, so I keep is out of scope.

Reply:
As an intermediate solution, I can add `getTree` call to `new_tab` and return the full `context_object`, but no of the data except `context_id` is needed, so it seems odd.